assets/javascripts/discourse/components/discourse-post-event/invitees.gjs

```diff
@@ -41,12 +41,14 @@ export default class DiscoursePostEventInvitees extends Component {
           </div>
         </section>
       {{else}}
-        <section class="event__section event-invitees no-rsvp">
-          <p class="no-rsvp-description">{{i18n
-              "discourse_post_event.models.invitee.status.going_count.other"
-              count="0"
-            }}</p>
-        </section>
+        {{#unless @event.isStandalone}}
+          <section class="event__section event-invitees no-rsvp">
+            <p class="no-rsvp-description">{{i18n
+                "discourse_post_event.models.invitee.status.going_count.other"
+                count="0"
```

Comment:
Is this `count="0"` correct?

Reply:
That is correct, this only appears when there are no respondents.